package.json

```diff
@@ -62,14 +62,13 @@
     "eslint": "8.18.0",
     "eslint-config-standard": "17.0.0",
     "eslint-config-standard-jsx": "11.0.0",
+    "eslint-import-resolver-webpack": "0.13.2",
```

Comment:
do we still need that without the webpack ? :)

Reply:
wahh

good catch thanks, removed